lib/Conch/Controller/Login.pm

```diff
@@ -305,19 +305,6 @@ sub session_logout ($c) {
     $c->status(204);
 }
 
-=head2 reset_password
-
-Resets a user's password, given an email address, and sends the user an email
-with their new password.
-
-=cut
-
-sub reset_password ($c) {
-    my $input = $c->validate_input('ResetPassword');
```

Comment:
Please also remove the json schema since it will no longer be used.

Reply:
So yeah my suggestion in the original ticket that we re-direct to `/user/me/password` was entirely wrong since the request bodies for those two endpoints are entirely different.

It's purged in the most recent rebase.